Extension/src/common.ts

```diff
@@ -888,6 +888,16 @@ export function promptReloadWindow(message: string): void {
     });
 }
 
+export function promptForIgnoreSettingsChange(): void {
+    promptIgnoreChange(localize("ignore.workspace.for.changes", "Setting change is ignored at folder level"));
```

Comment:
Since there's only one setting that can trigger this right now, it might be more useful to the customer to tell them specifically which setting is ignored.

Reply:
Not relevant due to scope change, has been removed